### PR TITLE
Update PhpDoc and small fixes

### PR DESCRIPTION
**Pull request by hacfi**
Some PhpDoc and code fixes

### CODE REVIEW

File: src/Docker/Container.php [hunk 1]
```diff
@@ -223,7 +223,7 @@ public function getExitCode()
             return $this->exitCode;
         }
 
-        if (is_array($this->runtimeInformations)) {
```
Comment (hacfi) [inline src/Docker/Container.php]: Throws an `Notice: Undefined index: State` otherwise.

File: src/Docker/Docker.php [hunk 2]
```diff
@@ -17,24 +16,18 @@
  */
 class Docker
 {
-    const BUILD_VERBOSE = false;
-    const BUILD_QUIET = true;
-
-    const BUILD_CACHE = true;
-    const BUILD_NO_CACHE = false;
-
```
Comment (hacfi) [inline src/Docker/Docker.php]: unused

File: src/Docker/Docker.php [hunk 3]
```diff
@@ -80,7 +73,7 @@ public function getImageManager()
 
    /**
      * Show the docker components version information
-     * @return json object with version values
```
Comment (hacfi) [inline src/Docker/Docker.php]: Not a valid type (see http://www.phpdoc.org/docs/latest/references/phpdoc/types.html)

File: src/Docker/Manager/ImageManager.php [hunk 4]
```diff
@@ -5,12 +5,12 @@
 use Docker\Exception\ImageNotFoundException;
 use Docker\Exception\UnexpectedStatusCodeException;
 use Docker\Image;
-use GuzzleHttp\Client;
```
Comment (hacfi) [inline src/Docker/Manager/ImageManager.php]: To be consistent with ContainerManager and as there is a Docker client as well it’s more clear.

**Comment by ubermuda** [comment]
Wow amazing! Thanks.

**Pull request merged**